projects/plugins/jetpack/_inc/client/recommendations/index.jsx

```diff
@@ -4,7 +4,8 @@
 import React from 'react';
 import { Redirect, Route, Switch } from 'react-router-dom';
 import { connect } from 'react-redux';
-import { __ } from '@wordpress/i18n';
+import { __, _x } from '@wordpress/i18n';
```

Comment:
you may like to fix this isse

Reply:
Addressed in 0d26021aeb7fd199366343ebfd7cdeaf4b403232